include/cppflow/tensor.h

```diff
@@ -205,7 +206,7 @@ namespace cppflow {
 
         // Convert to correct type
         const auto T_data = static_cast<T*>(raw_data);
```

Comment:
I have not tested it yet, but `res_tensor` is a temporary variable. Returning a data pointer from a temporary variable is alarming for me. Can we use `this->tf_tensor` to hold the resolved concrete tensor? Does it deallocate the original tf_tensor and corrupt the data if `this->tfe_handle` is created from it?

Reply:
Yeah, you are right! It indeed confuses people. Actually, `res_tensor` and `this->tf_tensor` hold the same buffer, but sometimes res_tensor uses a new buffer.

I think we should use the raw data of this->tf_tensor but not res_tensor.